rapids-4-spark-tools/README.md

```diff
@@ -1,78 +1,197 @@
+### Use from spark-submit
+```bash
+$SPARK_HOME/bin/spark-submit --class com.nvidia.spark.rapids.tool.qualification.QualificationMain \
+<Spark-Rapids-Repo>/rapids-4-spark-tools/target/rapids-4-spark-tools_2.12-<version>.jar \
+/path/to/eventlog1 /path/to/eventlog2
+```
+
+### Options
+```bash
+$SPARK_HOME/bin/spark-submit \
+--class com.nvidia.spark.rapids.tool.qualification.QualificationMain \
+<Spark-Rapids-Repo>/rapids-4-spark-tools/target/rapids-4-spark-tools_2.12-<version>.jar \
+--help
+
+For usage see below:
+
+  -i, --include-exec-cpu-percent   Include the executor CPU time percent. It
+                                   will take longer with this option and you may
+                                   want to limit the number of applications
+                                   processed at once.
+  -n, --num-output-rows  <arg>     Number of output rows for each Application.
+                                   Default is 1000.
+  -o, --output-directory  <arg>    Base output directory. Default is current
+                                   directory for the default filesystem. The
+                                   final output will go into a subdirectory
+                                   called rapids_4_spark_qualification_output.
+                                   It will overwrite any existing directory with
+                                   the same name.
+      --output-format  <arg>       Output format, supports csv and text. Default
+                                   is csv. text output format creates a file
+                                   named rapids_4_spark_qualification.log while
+                                   csv will create a file using the standard
+                                   Spark naming convention.
+  -h, --help                       Show help message
+
+ trailing arguments:
+  eventlog (required)   Event log filenames(space separated) or directories
+                        containing event logs. eg: s3a://<BUCKET>/eventlog1
+                        /path/to/eventlog2
+```
 
 ### Output
-By default this outputs a file `./rapids_4_spark_qualification.log` that contains the rankings of the applications. The output
-location can be changed using the `--output-directory` option.  Run `--help` for more information.
+By default this outputs a csv file under sub-directory `./rapids_4_spark_qualification_output/` that contains 
+the rankings of the applications.
+
+The output location can be changed using the `--output-directory` option. Default is current directory.
+
+The output format can be changed using the `--output-format` option. Default is csv.
+
+Run `--help` for more information.
+
+### Functions
+
+The qualification tool's algorithm is to calculate the dataframe SQL duration ratio = 
+sum(dataframe-SQL-duration) / application-duration
+and then rank all applications based on that ratio.
+
+Note: duration is in milli-seconds.
+
+It can also print the potential problems such as UDF.
```

Comment:
we should put more discalimer here and why its listed.  it won't catch all udfs and udfs can be handled but take additional steps, see the UDF sections in our docs

Reply:
Just added a notes for UDF. Could you review again?thx